### PR TITLE
Minor tweaks and tests refactors in neptune.cli

### PR DESCRIPTION
**Pull request by Raalsky**
## Before submitting checklist

- [ ] Did you **update the CHANGELOG**? (not for test updates, internal changes/refactors or CI/CD setup)
- [ ] Did you **ask the docs owner** to review all the user-facing changes?

### CODE REVIEW

File: src/neptune/cli/clear.py [hunk 1]
```diff
@@ -87,11 +89,17 @@ def remove_data(
...
     def remove_containers(paths: List[Path]) -> None:
         for path in paths:
-            try:
-                shutil.rmtree(path)
-                logger.info(f"Deleted: {path}")
-            except OSError:
-                logger.warn(f"Cannot remove directory: {path}")
+            if path.exists():
+                try:
+                    shutil.rmtree(path)
```
Comment (artsiomtserashkovich) [inline src/neptune/cli/clear.py:102]: What do you think about make it the same as https://github.com/neptune-ai/neptune-client/blob/fe39583f7aa580fa040f5b709946abcdeedae9ca/src/neptune/cli/clear.py#L64? (``ignoreErrors=True`` or left only one implementation)

Comment (Raalsky) [reply src/neptune/cli/clear.py:102]: That's completely different thing. Sync that's an Experiment (run, model etc.) created in `sync` mode (every operation = immediate request to server) vs async that wasn't cleaned up properly (mode `async` that synced all the operations but there are leftovers)

Comment (artsiomtserashkovich) [reply src/neptune/cli/clear.py:102]: In that particular case, I meant diff at error handling - in one case we are using the bultin flag to suppress and in another, we are using try...except -> what do you think about make it the same?

Comment (Raalsky) [reply src/neptune/cli/clear.py:102]: Hmmmm. We probably wanted here (I've copied and pasted it from some other cleanups) to be more explicit. With `sync` we're not losing anything etc and that's a "bulk" delete of all containers so quite different usage.

**Review by artsiomtserashkovich — APPROVED** [review]

**Pull request merged**